### PR TITLE
Fix additional comments from Raphael

### LINKED ISSUE #65: Unclear how a sample makes its way from a platform collector to a ComputePressureUpdateCallback
https://wicg.github.io/compute-pressure/#platform-primitives says

> When a sample is obtained from the [platform collector](https://wicg.github.io/compute-pressure/#dfn-platform-collector), the [user agent](https://wicg.github.io/compute-pressure/#dfn-user-agent) MUST run the [8.6.4 Notify Compute Pressure Observers](https://wicg.github.io/compute-pressure/#notify-observers) algorithm a per the [8.6 Processing Model](https://wicg.github.io/compute-pressure/#processing-model)

That looks like the wrong algorithm though, as the process through which the sample collected by the platform is added to a `ComputePressureRecord`, how this `ComputePressureRecord` is created and how the relevant `ComputePressureObserver` instances are selected for a given target type are not clear.

### PR DESCRIPTION
**Pull request by kenchris**
Closes #65

***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/compute-pressure/pull/69.html" title="Last updated on Mar 2, 2022, 1:26 PM UTC (df53f64)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/compute-pressure/69/a8ac1f7...kenchris:df53f64.html" title="Last updated on Mar 2, 2022, 1:26 PM UTC (df53f64)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -610,8 +610,8 @@ <h3>Processing Model</h3>
   <section>
     <h3>Queue a ComputePressureRecord</h3>
     <p>
-      To <dfn>queue telemetry data</dfn> given the arguments |target| and |data|,
-      run these steps:
+      To <dfn>queue telemetry data</dfn> given the arguments |observer:ComputePressureObserver|,
```
Comment (rakuco) [inline index.html:613]: Now you need to update https://wicg.github.io/compute-pressure/#platform-primitives and make the call to "queue telemetry data" there also pass an `observer`. The question is what observer you'd be referring to in that section.

File: index.html [hunk 2]
```diff
@@ -644,6 +638,13 @@ <h3>Create and populate a ComputePressureRecord</h3>
       as thermal conditions.
     </aside>
     <ol class="algorithm">
+      <li>
+        Let |document:Document| be the [=environment settings object/responsible document=]
+        of the [=current settings object=].
+      </li>
+      <li>
+        If |document| is not {{undefined}} and is not [=Document/fully active=], about these steps.
```
[the remark this answers is not in the document]
Comment (kenchris) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (kenchris) [reply index.html:614]: Right, fixed

**Review by rakuco — APPROVED** [review]
lgtm!

**Pull request merged**